app/src/main/java/com/mobiledevelopmentfoodapp/dto/Customer.kt

```diff
@@ -1,10 +1,12 @@
 package com.mobiledevelopmentfoodapp.dto
 
-//import androidx.room.Entity
-//import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Customer(var name: String, var phone: String, @SerializedName("id") var customerId:Int = 0 ) {
+data class Customer(
```

Comment:
Formatting and visibility modifiers. Visibility modifier best practices can be found at https://kotlinlang.org/docs/visibility-modifiers.html Formatting can be found at https://kotlinlang.org/docs/coding-conventions.html#formatting